subprojects/packagefiles/libkqueue/meson.build

```diff
@@ -144,7 +144,8 @@ libkqueue = library(
   dependencies: kqueue_deps,
 )
 
+dep_inc = include_directories('include')
 libkqueue_dep = declare_dependency(
   link_with: libkqueue,
-  include_directories: kqueue_inc,
+  include_directories: dep_inc,
```

Comment:
This could just be `include_directories: 'include'`

Reply:
sure, OTOH this was a pattern for compatibility with 0.49.